exercism/bracket-push/bracket-push.js

```diff
@@ -0,0 +1,31 @@
+function bracket(bracketString) {
+    if (bracketString.match(/[\{}()[\]]/ig) && bracketString.length%2 === 0){
+      if(bracketString[0] == '{' && bracketString[bracketString.length-1] == '}' && bracketString.match(/[{]/ig).length === bracketString.match(/[}]/ig).length){
```

Comment:
Have long lines split into different lines of code. This makes it easier to read.